server/tests/models/article.spec.js

```diff
@@ -4,45 +4,45 @@ import chaiAsPromise from 'chai-as-promised';
 import models from '<serverModels>';
 import { createEllipsis } from '<helpers>/utils';
 import { newArticle } from '<fixtures>/article';
+import newCategory from '<fixtures>/category';
 
 chai.use(chaiAsPromise);
 const { expect } = chai;
-const { Article, sequelize } = models;
-
+const { Article, Category, sequelize } = models;
+let article;
+let category;
+let newArticleData;
 before(async () => {
   try {
     await sequelize.sync({ force: true });
+    const articleCategory = { ...newCategory, categoryName: 'technology' };
```

Comment:
change categoryName to name, also why are you spreading an object with one property that you are already overwriting?